base/utils.sh

```diff
@@ -45,12 +45,24 @@ set_nginx_port() {
     # the nginx s6 service templates/copies the file from /nginx/nginx.conf.j2;/nginx/nginx.conf,
     # which overwrites our changes, so we have to change the source file.
     if [[ -f /nginx/nginx.conf ]]; then
-      /usr/bin/sed -i s/listen\ 80/listen\ "${NGINX_PORT}"/g /nginx/nginx.conf
-      /usr/bin/sed -i s/listen\ 443/listen\ "${NGINX_SSL_PORT}"/g /nginx/ssl_nginx.conf
+      echo "NGINX.CONF before"
+      grep listen /nginx/nginx.conf
+      echo
+      /usr/bin/sed -i s/\}\}80/\}\}"${NGINX_PORT}"/g /nginx/nginx.conf
+      /usr/bin/sed -i s/\}\}443/\}\}"${NGINX_SSL_PORT}"/g /nginx/ssl_nginx.conf
+      echo "NGINX.CONF after"
+      grep listen /nginx/nginx.conf
+      echo
```

Comment:
You don't need to modify this. This is for an even older version of the base image.